hibernate-core/src/main/java/org/hibernate/cfg/Settings.java

```diff
@@ -194,7 +194,7 @@ public boolean isJtaTrackByThread() {
 	}
 
 	public boolean isStrictJPAQLCompliance() {
-		return sessionFactoryOptions.isStrictJpaQueryLanguageCompliance();
+		return sessionFactoryOptions.getJpaCompliance().isJpaCacheComplianceEnabled();
```

Comment:
@jrenaat pretty sure this should have been `isJpaQueryComplianceEnabled()` instead of `isJpaCacheComplianceEnabled()`, but don't worry: nothing calls this method so I think I'm just going to remove it.